src/js/models/v2/awards/financialSystemDetails/BaseFinancialSystemDetailsRow.js

```diff
@@ -12,8 +12,10 @@ export const formatDate = (date) => date.format('MM/DD/YYYY');
 const BaseFinancialSystemDetailsRow = {
     populate(data) {
         this.id = data.financial_accounts_by_awards_id || '';
+        this.reporting_fiscal_year = data.submission.reporting_fiscal_year || null;
+        this.reporting_fiscal_quarter = data.submission.reporting_fiscal_quarter || null;
```

Comment:
First check for the existence of `data.submission` to prevent a `Cannot read property of undefined` error in case the submission object isn't present in the response. You can see examples of where we do this for `data.treasury_account`, etc. below.